src/Grapevine/Server/RestServer.cs

```diff
@@ -172,7 +170,24 @@ public string Port
             }
         }
 
-        public IPublicFolder PublicFolder { get; }
+        public IPublicFolder PublicFolder
+        {
+            get
+            {
+                if (!PublicFolders.Any()) PublicFolders.Add(new PublicFolder());
+                return PublicFolders.First();
```

Comment:
This is here purely for backwards compatibility. It will be removed in a future release, I just haven't decided where.

Also, some users have asked that using the PublicFolders feature be completely optional.